### PR TITLE
Foodcritic/rubocop test / Debian support

### PR DESCRIPTION
**Pull request by goacid**
Just few changes to add debian support.
Require ruby 2.2 (should be an attribut)

### CODE REVIEW

**Comment by ryansb** [comment]
:+1: This looks good for me. @tysonstewart can you take a look?

**Comment by tysonstewart** [comment]
I don't do much with these anymore. Let me pull in someone more knowledgeable.

File: recipes/install.rb [hunk 1]
```diff
@@ -24,10 +24,24 @@
...
 
 case node['platform_family']
-when "rhel","fedora"
-    include_recipe 'yum::epel'
-    execute "Install riemann" do
-        not_if {File.exists?("/usr/bin/riemann")}
-        command "yum localinstall -y http://aphyr.com/riemann/riemann-#{node[:riemann][:version]}-1.noarch.rpm"
-    end
+when 'rhel', 'fedora'
+  include_recipe 'yum''epel'
```
Comment (chrisgilbert) [inline recipes/install.rb]: Is this correct?

**Comment by chrisgilbert** [comment]
Just one line note on there - not sure if ```include_recipe 'yum''epel'``` is correct?  Everything else looks good to me :+1:

**Comment by goacid** [comment]
My bad, should be 'yum::epel'

**Comment by chrisgilbert** [comment]
I've merged now to master, thanks so much for your contribution :+1:

**Pull request merged**